src/Illuminate/Auth/Notifications/VerifyEmail.php

```diff
@@ -58,14 +65,29 @@ public function toMail($notifiable)
      */
     protected function verificationUrl($notifiable)
     {
-        return URL::temporarySignedRoute(
-            'verification.verify',
-            Carbon::now()->addMinutes(Config::get('auth.verification.expire', 60)),
-            [
-                'id' => $notifiable->getKey(),
-                'hash' => sha1($notifiable->getEmailForVerification()),
-            ]
-        );
+        if (static::$createUrlCallback) {
+            return call_user_func(static::$createUrlCallback, $notifiable);
+        } else {
```

Comment:
I think not needed "else" in here.

Reply:
Look: https://github.com/laravel/framework/blob/aada574a3929d043d2e55820f6fefa85f2ae8e23/src/Illuminate/Auth/Notifications/ResetPassword.php#L64

Reply:
I just kept it the same as it is written in `ResetPassword::toMail`, hence the `else`. I agree that it is not necessary...